chapters/ch05/2.index-datasets.ipynb

```diff
@@ -9,20 +9,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import sys\n",
-    "\n",
-    "sys.path.append('../..')\n",
+    "from pyspark.sql import SparkSession\n",
     "\n",
     "from aips import get_engine\n",
     "from aips.spark.dataframe import from_csv\n",
-    "from pyspark.sql import SparkSession\n",
     "\n",
     "spark = SparkSession.builder.appName(\"AIPS\").getOrCreate()\n",
-    "engine = get_engine()"
+    "engine = get_engine(\"solr\")"
```

Comment:
Not sure if this needs to be committed like this for now and removed later once we have embedded/local Solr or if this should be corrected now, but just noting the hard-coded engine since it looks suspect. I'm guessing this was done since this data is only for the SKG.

Reply:
I've implemented dual indexing for Ch5, allowing the removal of hard-coding solr